.pipelines/containerSourceData/scripts/BuildBaseContainers.sh

```diff
@@ -106,10 +114,16 @@ function validate_inputs {
     DISTROLESS_BASE_TARBALL=$(find "$CONTAINER_TARBALLS_DIR" -name "distroless-base-[0-9.]*.tar.gz")
     DISTROLESS_DEBUG_TARBALL=$(find "$CONTAINER_TARBALLS_DIR" -name "distroless-debug-[0-9.]*.tar.gz")
     DISTROLESS_MINIMAL_TARBALL=$(find "$CONTAINER_TARBALLS_DIR" -name "distroless-minimal-[0-9.]*.tar.gz")
-    if [[ (! -f $BASE_TARBALL)  || \
-        (! -f $DISTROLESS_BASE_TARBALL) || \
-        (! -f $DISTROLESS_DEBUG_TARBALL) || \
-        (! -f $DISTROLESS_MINIMAL_TARBALL) ]]; then
+    #give default values
+    BASE_BUILD=${BASE_BUILD:-true}
+    DISTROLESS_BASE_BUILD=${DISTROLESS_BASE_BUILD:-true}
+    DISTROLESS_DEBUG_BUILD=${DISTROLESS_DEBUG_BUILD:-true}
+    DISTROLESS_MINIMAL_BUILD=${DISTROLESS_MINIMAL_BUILD:-true}
+
+    if [[ ($BASE_BUILD == true && ! -f $BASE_TARBALL) || \
+      ($DISTROLESS_BASE_BUILD == true && ! -f $DISTROLESS_BASE_TARBALL) || \
+      ($DISTROLESS_DEBUG_BUILD == true && ! -f $DISTROLESS_DEBUG_TARBALL) || \
+      ($DISTROLESS_MINIMAL_BUILD == true && ! -f $DISTROLESS_MINIMAL_TARBALL) ]]; then
```

Comment:
I think these booleans are coming from the YAML booleans which, for reasons beyond my understanding, are not consistent it terms of using upper or lower case. Considering that, I'd fortify these checks to accept both `True` and `true`. See how `PUBLISH_TO_ACR` is read in this file to see what I mean.

Reply:
updated